src/Sulu/Bundle/MediaBundle/DependencyInjection/SuluMediaExtension.php

```diff
@@ -277,13 +277,15 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('services_imagine_svg.xml');
         }
 
-        if (\class_exists(VipsImagine::class)) {
+        $hasVipsAdapter = false;
+        if (\class_exists(VipsImagine::class) && \extension_loaded('vips')) {
             $loader->load('services_imagine_vips.xml');
+            $hasVipsAdapter = true;
         }
 
         if ('auto' === $config['adapter']) {
             $adapter = 'gd';
-            if (\class_exists(VipsImagine::class)) {
+            if ($hasVipsAdapter) {
                 $adapter = 'vips';
             } elseif (\class_exists(\Imagick::class)) {
                 $adapter = 'imagick';
```

Comment:
As `\Imagick` is part of the extension there works there with only this check as the imagine imagick adapter is part of imagine core library. For vips we need to check for both existing of the imagine adapter and existing of the php extension.